src/components/form-group/form-group.js

```diff
@@ -386,11 +387,20 @@ export default (resolve, reject) => {
           if (input) {
             const adb = 'aria-describedby'
             let ids = (getAttr(input, adb) || '').split(/\s+/)
+            add = (add || '').split(/\s+/)
             remove = (remove || '').split(/\s+/)
             // Update ID list, preserving any original IDs
+            // and ensuring the ID's are unique
             ids = ids
               .filter(id => !arrayIncludes(remove, id))
-              .concat(add || '')
+              .concat(add)
+              .filter(Boolean)
+            ids = keys(
+              ids.reduce((memo, id) => {
```

Comment:
`ids.reduce((result, id) => { ...result, [id]: true }, {})`

Reply:
how about (since objects being returned by arrow function shorthand need to be surrounded in brackets):
```js
ids.reduce((result, id) => ({ ...result, [id]: true }), {})
```